Async validators: Argument of type '{ asyncValidators: { name: string; validation: (control: FormControl) => any; }[]; }' is not assignable to parameter of type 'ConfigOption

<!--
IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION WE MIGHT CLOSE YOUR ISSUE WITHOUT INVESTIGATING
-->

**I'm submitting a ...** 
```
[ ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ X] support request
```

**Current behavior**
I'm getting an error trying to add asyncValidators

```
error TS2345: Argument of type '{ asyncValidators: { name: string; validation: (control: FormControl) => any; }[]; }' is not assignable to parameter of type 'ConfigOption'.
      Object literal may only specify known properties, and 'asyncValidators' does not exist in type 'ConfigOption'.
```

**Minimal reproduction of the problem with instructions**
Guides I followed: https://ngx-formly.github.io/ngx-formly/guide/validation

I am importing Formly in my module as in the guides. 
And declaring `asyncValidators`. But it seems that `asyncValidators` is not a valid object key to add to `ConfigOption`.


```
export function ipAsyncValidator(control: FormControl): Promise {
  return new Promise((resolve, reject) => {
    setTimeout(() => {
      if (/(\d{1,3}\.){3}\d{1,3}/.test(control.value)) {
        resolve({ 'ip': true });
      } else {
        resolve(null);
      }
    }, 1000);
  });
}
```

```
FormlyModule.forRoot({
      asyncValidators: [
        { name: 'ipAsync', validation: ipAsyncValidator }
      ]
    }),
```
It's actually true, that `ConfigOption` does not have any `asyncValidators` key.

```
export interface ConfigOption {
    types?: TypeOption[];
    wrappers?: WrapperOption[];
    validators?: ValidatorOption[];
    extensions?: ExtensionOption[];
    validationMessages?: ValidationMessageOption[];
    /** @deprecated use `extensions` instead */
    manipulators?: ManipulatorOption[];
    extras?: {
        /** @deprecated use `extensions` instead */
        fieldTransform?: any;
        immutable?: boolean;
        showError?: (field: FieldType) => boolean;
        /**
         * Defines the option which formly rely on to check field expression properties.
         * - `modelChange`: perform a check when the value of the form control changes.
         * - `changeDetectionCheck`: triggers an immediate check when `ngDoCheck` is called.
        */
        checkExpressionOn?: 'modelChange' | 'changeDetectionCheck';
    };
}
```

I'm using Angular 8 and `@ngx-formly/core` 5.4.1.

I'm probably making something bad.

Could you help me, please?

Thanks :)

--- a/demo/src/app/examples/advanced/json-schema/config.module.ts
+++ b/demo/src/app/examples/advanced/json-schema/config.module.ts
@@ -23,6 +23,7 @@ import { AppComponent } from './app.component';
               { file: 'app.module.ts', content: require('!!highlight-loader?raw=true&lang=typescript!./app.module.ts'), filecontent: require('!!raw-loader!./app.module.ts') },
               { file: 'array.type.ts', content: require('!!highlight-loader?raw=true&lang=typescript!./array.type.ts'), filecontent: require('!!raw-loader!./array.type.ts') },
               { file: 'object.type.ts', content: require('!!highlight-loader?raw=true&lang=typescript!./object.type.ts'), filecontent: require('!!raw-loader!./object.type.ts') },
+              { file: 'null.type.ts', content: require('!!highlight-loader?raw=true&lang=typescript!./null.type.ts'), filecontent: require('!!raw-loader!./null.type.ts') },
               { file: 'assets/json-schema/simple.json', content: require('!!highlight-loader?raw=true&lang=typescript!./assets/simple_json'), filecontent: require('!!raw-loader!./assets/simple_json') },
               { file: 'assets/json-schema/nested.json', content: require('!!highlight-loader?raw=true&lang=typescript!./assets/nested_json'), filecontent: require('!!raw-loader!./assets/nested_json') },
               { file: 'assets/json-schema/arrays.json', content: require('!!highlight-loader?raw=true&lang=typescript!./assets/arrays_json'), filecontent: require('!!raw-loader!./assets/arrays_json') },

--- a/demo/src/app/guides/validation.md
+++ b/demo/src/app/guides/validation.md
@@ -56,26 +56,6 @@ export function IpValidator(control: FormControl): ValidationErrors {
 })
 ```
 
-**ALERT!** If your function is async, you need to include it within `asyncValidators` property:
-```typescript
-export function ipAsyncValidator(control: FormControl): Promise {
-  return new Promise((resolve, reject) => {
-    setTimeout(() => {
-        if(/(\d{1,3}\.){3}\d{1,3}/.test(control.value)) {
-              resolve({ 'ip': true });
-        } else {
-              resolve(null);
-        }
-    }, 1000);
-  });
-}
-...
-FormlyModule.forRoot({
-  asyncValidators: [
-    { name: 'ipAsync', validation: ipAsyncValidator },
-  ],
-})
-```
 ##### FIELD WITH CUSTOM VALIDATION
 You just need to include the name of the validate function, declared in `FormlyModule`, within the property `validators.validation`.
 ```typescript

